presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -268,6 +268,40 @@ and a file system location of ``s3://test_bucket/test_schema/test_table``:
         location = 's3://test_bucket/test_schema/test_table')
     )
 
+Caching Support
+----------------
+
+Manifest File Caching
+^^^^^^^^^^^^^^^^^^^^^^
+
+Starting from Iceberg version 1.1.0, Apache Iceberg provides a mechanism to cache the contents of Iceberg manifest files in memory. This manifest caching feature helps
+to reduce repeated reads of small Iceberg manifest files from remote storage.
+
+.. note::
+
+    Currently, manifest file caching is supported for Hadoop and Nessie catalogs in the Presto Iceberg connector.
```

Comment:
@ChunxuTang As of now from Presto no. Since manifest file caching is supported with `HadoopFileIO` in Iceberg. But in Presto for Hive Catalog we have custom [HdfsFileIO](https://github.com/prestodb/presto/blob/master/presto-iceberg/src/main/java/com/facebook/presto/iceberg/HdfsFileIO.java) 

Do we know why do we use this custom HdfsFileIO from Presto for Iceberg Hive Catalog?